### ISSUE TITLE
Badly named configuration option

### ISSUE THREAD

**Issue by hasghari** [aasm/aasm#192]
The configuration option `skip_validation_on_save` does not correspond to the expected behavior. I was expecting a `save(validate: false)` when that option is `true` but instead it is completely bypassing all calbacks and only persisting the state column.

I would suggest either changing the behavior to match the name or changing the name (`only_persist_state_on_save`) to match the behavior as to avoid confusion. In either case, this change would not be backwards compatible so I'm not sure how best to deal with that.

**Comment by throttleup** [comment]
Possibly separate issue. bang methods don't seem to transition new_records when `skip_validation_on_save` is true. It will set the value back to `old_value` as the `update_all` won't match any records `where(self.class.primary_key => self.id)`

**Comment by alto** [comment]
@throttleup Yep, that sounds like a bug. Could you please open another issue for that?

**Comment by alto** [comment]
@throttleup Is that #242 then?

**Comment by throttleup** [comment]
it is

**Comment by anilmaurya** [comment]
`only_persist_state_on_save` makes more sense for this behaviour . :+1:  for this.
